src/modules/Checkout/CheckOut.tsx

```diff
@@ -0,0 +1,18 @@
+import CartBanner from "../ShoppingCart/Components/CartBanner";
+import OrderDetails from "./Components/OrderDetails";
+import PropTypes from "prop-types";
+
+const CheckOut = ({ activeState }: { activeState: number }) => {
+  return (
+    <div>
+      <CartBanner activeState={activeState} />
+      <OrderDetails />
+    </div>
+  );
+};
+
+CheckOut.propTypes = {
```

Comment:
In latest React development `.propTypes` is not needed and also you are using typescript already in your application